packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -25,11 +28,31 @@ const testContainerConfig: ITestContainerConfig = {
     registry: [["sharedString", SharedString.getFactory()]],
 };
 
+class MockDetachedBlobStorage implements IDetachedBlobStorage {
+    private readonly blobs = new Map<number, ArrayBufferLike>();
+    private blobCount = 0;
+
+    public async createBlob(content: ArrayBufferLike): Promise<ICreateBlobResponse> {
+        this.blobs.set(++this.blobCount, content);
+        return {
+            id: this.blobCount.toString(),
+            url: "",
```

Comment:
I thought Daniel raised PR earlier to remove URL. Can we kill it? :)

Reply:
Hasn't been released yet. I can leave it out since it isn't used but it'd need a type assertion